core/trino-main/src/test/java/io/trino/server/security/TestResourceSecurity.java

```diff
@@ -141,7 +153,7 @@ public void testInsecureAuthenticatorHttp()
                         .put("http-server.authentication.insecure.user-mapping.pattern", ALLOWED_USER_MAPPING_PATTERN)
                         .build())
                 .build()) {
-            server.getInstance(Key.get(AccessControlManager.class)).addSystemAccessControl(new TestSystemAccessControl());
+            server.getInstance(Key.get(AccessControlManager.class)).addSystemAccessControl(new TestSystemAccessControl(false));
```

Comment:
Maybe add a static factory since the true/false makes it hard to tell what is happening